pandas_datareader/yahoo/daily.py

```diff
@@ -97,7 +98,9 @@ def yurl(symbol):
 
     def _get_params(self, symbol):
         unix_start = int(time.mktime(self.start.timetuple()))
-        unix_end = int(time.mktime(self.end.timetuple()))
+        day_end = self.end + datetime.timedelta(days=1) \
```

Comment:
It is a datetime created from year/month/day (as discussed in https://github.com/pydata/pandas-datareader/issues/356 there still might be some timezone issues ...)
The yahoo API becoming very sensitive ...